android/app/build.gradle

```diff
@@ -56,7 +56,31 @@ project.ext.react = [
     enableHermes : true,
 ]
 
+def signingDebug = getSigningConfigs()['debug']
```

Comment:
Can we replace all the single-quotes with doubles? When we migrate to the Kotlin DSL later, it will require all strings to be double-quoted.

Reply:
thanks! done